### ISSUE TITLE
Survey List UI Changes

### ISSUE THREAD

**Issue by willdoran** [ushahidi/platform#2522]
### Overview
Add new UI for Survey creation, preserve but hide existing UI based on setting of feature flag

### Requirements
- [ ] New UI should only be displayed if the SMS Beta Feature flag is enabled
- [ ] Hide yellow button when SMS Beta Feature flag is enabled
- [ ] Add Create Survey section

### Dependencies
- #

### Test Script
- [ ] When SMS Beta Feature Flag is disabled 
  - [ ] Yellow button is visible
  - [ ] Yellow button when clicked changes view to Survey creation view
  - [ ] Create Survey section is not visible
- [ ] When SMS Beta Feature Flag is enabled 
  - [ ] Yellow button is hidden
  - [ ] Create Survey section is visible
  - [ ] When New Survey button is clicked changes view to Survey creation view
  - [ ] View appears as below

![Screen Shot 2018-02-27 at 13.59.42.png](https://images.zenhubusercontent.com/5a6a44264b5806bc2bc4546d/b4bc1cc9-a6b2-4135-abcd-7151765b6fe3)

**Comment by rowasc** [comment]
@willdoran this doesn't sound right. The new survey button should still be visible even when not feature flagged for SMS (it's just the style that changed), the only difference is if you see the sidebar link to create a targeted SMS survey after you start creating a new survey (https://xd.adobe.com/view/a9b0da01-2589-465d-ab05-db3a178d6a6f/screen/50c775a7-ac35-495f-9a35-1d62c4d118ec/New-Survey-jank)

**Comment by rowasc** [comment]
Confirmed with Justin. The FAB goes away and the new survey button is the one that remains

**Comment by rowasc** [comment]
Needs review. 
@willdoran

**Comment by rowasc** [comment]
This no longer has a dependency which means it can be Ready for Dev as soon as it's reviewed

**Comment by crcommons** [comment]
Reminder to update the issue with changes. Hard to follow in the comments. Thanks :)

**Comment by Angamanga** [comment]
Findings from QA:
- Clicking on "Create survey"-text gives a 404

**Comment by StaicyG** [comment]
@Angamanga seems that the error 404 issue has been solved. Have a look to confirm..

**Comment by StaicyG** [comment]
@kinstelli @crcommons 

Passed QA testing

**Comment by StaicyG** [comment]
@kinstelli @crcommons @Angamanga 

Passed QA test on .io

**Comment by Eve-wanderi** [comment]
Passed QA
